src/main/java/org/apache/maven/plugins/dependency/utils/CopyUtil.java

```diff
@@ -78,12 +85,15 @@ public void copyArtifactFile(Artifact sourceArtifact, File destination) throws I
      *
      * @param source the source file to copy
      * @param destination the destination file
+     * @param silent if true, use debug log level, otherwise info for logging
      * @throws IOException if copy has failed
      *
      * @since 3.2.0
      */
-    public void copyFile(File source, File destination) throws IOException {
-        logger.debug("Copying file '{}' to {}", source, destination);
```

Comment:
I really can't see why this should be anything other than debug by default. This is not an actionable message.

Reply:
eg. When using `copy` mojo from cli I need an information what was copied for auditing